pygls/lsp/types/language_features/completion.py

```diff
@@ -136,27 +136,27 @@ class InsertReplaceEdit(Model):
 
 class CompletionItem(Model):
     label: str
-    kind: Optional[CompletionItemKind] = None
-    tags: Optional[List[CompletionItemTag]] = None
-    detail: Optional[str] = None
-    documentation: Optional[Union[str, MarkupContent]] = None
-    deprecated: Optional[bool] = False
-    preselect: Optional[bool] = False
-    sort_text: Optional[str] = None
-    filter_text: Optional[str] = None
-    insert_text: Optional[str] = None
-    insert_text_format: Optional[InsertTextFormat] = None
-    insert_text_mode: Optional[InsertTextMode] = None
-    text_edit: Optional[Union[TextEdit, InsertReplaceEdit]] = None
-    additional_text_edits: Optional[List[TextEdit]] = None
-    commit_characters: Optional[List[str]] = None
-    command: Optional[Command] = None
-    data: Optional[Any] = None
+    kind: Optional[CompletionItemKind]
+    tags: Optional[List[CompletionItemTag]]
+    detail: Optional[str]
+    documentation: Optional[Union[str, MarkupContent]]
+    deprecated: Optional[bool]
+    preselect: Optional[bool]
+    sort_text: Optional[str]
+    filter_text: Optional[str]
+    insert_text: Optional[str]
+    insert_text_format: Optional[InsertTextFormat]
+    insert_text_mode: Optional[InsertTextMode]
+    text_edit: Optional[Union[TextEdit, InsertReplaceEdit]]
+    additional_text_edits: Optional[List[TextEdit]]
+    commit_characters: Optional[List[str]]
+    command: Optional[Command]
+    data: Optional[Any]
 
 
 class CompletionList(Model):
     is_incomplete: bool
-    items: List[CompletionItem] = []
+    items: List[CompletionItem]
```

Comment:
Could you please check and initialize `items` filed in `add_item` and `add_items` methods?

Reply:
I don't _think_ this should be necessary.  How has the user constructed a `CompletionList` on which to call these methods without initializing the `items`?

Can you show me some code that goes wrong without the change that you suggest?

Reply:
I guess you are right, I am still not used to pydantic.